cucumber/policy/features/support/hooks.rb

```diff
@@ -1,9 +1,36 @@
 # frozen_string_literal: true
 
+# Generates random unique names
+#
+require 'haikunator'
+require 'fileutils'
+
 Before do |scenario|
   @scenario_name = scenario.name
 end
 
 Before "@echo" do |scenario|
   @echo = true
 end
+
+Before do
```

Comment:
Seems like a copy of another 2 files I saw earlier - can we just create a common module for them?

Reply:
Yes, we should, but I don't want to do it now.  Unless you know offhand the correct way to share hooks among profiles in rspec (rspec, making you relearn its own special way to do everything which would be simple in vanilla ruby).

My feeling is add a ticket.  I've been talking with Jason and Brian about doing a large overhaul of all this code.  These duplicated blocks are just the tip of a giant iceberg.

Reply:
https://github.com/cyberark/conjur/issues/1470